pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentRestletResource.java

```diff
@@ -209,6 +214,40 @@ public List<Map<String, Object>> getServerToSegmentsMap(
     return resultList;
   }
 
+  @GET
+  @Path("segments/{tableName}/xdebug")
+  @Produces(MediaType.APPLICATION_JSON)
+  @ApiOperation(value = "Debug co-location")
+    public Map<Integer, Set<String>> getCoLocationDebugInfo(
```

Comment:
Repurpose/remove this.. ideally we should be able to see table-group related metadata/stats in the group details page.